test/integration/full/controls/gsuite.rb

```diff
@@ -17,11 +17,6 @@
 group_role                  = attribute('group_role')
 project_id                  = attribute('project_id')
 service_account_email       = attribute('service_account_email')
-credentials_path            = attribute('credentials_path')
```

Comment:
Has this test fixture been tested when the group is being created? A service account credentials JSON file is required when creating a group due to the limitations with default application credentials. I suspect that this fixture will fall over if `create_group` is set to `true`.

Reply:
It took me a second to recall this - the `full` test fixture [still requires a credentials_path](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/6cfc2cc8593cf23f0d552252ba01ad8fc4a1c878/test/fixtures/full/extra_variables.tf#L17). We're dropping the `credentials_path` lookup here because this was just configuring a file for running `gcloud`. Because `gcloud` can be configured with the `CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE` environment variable and inspec-gcp can be configured with the `GOOGLE_APPLICATION_CREDENTIALS` environment variable, we can push the entire matter of credentials out of testing and into the environment.

So good catch on this, but since this is only dealing with tests, this code should work correctly in the `full` test fixture with `create_group = true`.